it.lproj/Localizable.strings

```diff
@@ -116,13 +116,13 @@
 "Automatic Distance Estimation" = "Stima automatica della distanza";
 
 /* Automatic Distance Explanation */
-"Distance Explanation" = "Le stime camminavano distanza automaticamente, questo è in genere più preciso rispetto all'utilizzo di una lunghezza di falcata fissa.";
+"Distance Explanation" = "La lunghezza della falcata viene calcolata automaticamente dai tuoi dispositivi, di solito è più precisa rispetto all'utilizzo di una lunghezza fissa.";
```

Comment:
Fixed context understanding error. Now the user understands whether to use the automatic calculation of the stride or to establish it as wishes.